charts/tidb-operator/templates/admission/admission-webhook-deployment.yaml

```diff
@@ -34,6 +34,14 @@ spec:
             - --tls-cert-file=/var/serving-cert/cert.pem
             - --tls-private-key-file=/var/serving-cert/key.pem
             {{- end }}
+          livenessProbe:
```

Comment:
should be readinessProbe? actually we need both I suppose

Reply:
my mistake, fixed